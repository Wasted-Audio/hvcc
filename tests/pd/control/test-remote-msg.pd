#N canvas 591 80 450 300 12;
#X obj 281 37 loadbang;
#X obj 56 108 r bla;
#X obj 56 175 print;
#X obj 104 107 r die;
#X obj 156 108 r blub;
#X msg 281 72 3 2;
#X obj 251 38 bng 20 250 50 0 empty empty empty 0 -10 0 12 #fcfcfc #000000 #000000;
#X msg 281 108 _ \; bla 1 \; die \$1 2 \; blub 4 \, \$2 1;
#X connect 0 0 5 0;
#X connect 1 0 2 0;
#X connect 3 0 2 0;
#X connect 4 0 2 0;
#X connect 5 0 7 0;
#X connect 6 0 5 0;
