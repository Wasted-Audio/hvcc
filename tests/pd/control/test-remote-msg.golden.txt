[@ 0.000] print: 1
[@ 0.000] print: 3 2
[@ 0.000] print: 4
[@ 0.000] print: 2 1
